docs/resources/json.md.erb

```diff
@@ -47,6 +47,21 @@ where
 * `name` is a configuration setting in a JSON file
 * `should eq 'foo'` tests a value of `name` as read from a JSON file versus the value declared in the test
 
+The `json` resource can also be used with json formatted output from a command. 
```

Comment:
Could you capitalize to `JSON` when referring to the file format?

Reply:
DONE 😁